README.md

```diff
@@ -1 +1,245 @@
+  - [Usage - Conference Page of Web Summit](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment on Git Hub](#triangular_flag_on_post-deployment)
+- [👥 Author: Baqar Abbas](#authors)
+- [🔭 Future Features will be updated and enhanced](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Conference Site Project] <a name="about-project"></a>
+
+
+
+**[Conference Site Project ]** is my first Microverse Portfolio Project...
+
+## 🛠 Built With HTMl, CSS and Javascript on Visual studio Code in collaboration with Git Hub <a name="built-with"></a>
+
+### Tech Stack HTML, CSS and JavaScript using VS CODE and Git Hub<a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+  Built on HTML, CSS and Javascript using Visual studio Code in Collaboration with Git Hub 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.google.com/">HTML</a></li>
+    <li><a href="https://www.google.com/">CSS</a></li>
+    <li><a href="https://www.google.com"">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features demostration of Conference Page <a name="key-features"></a>
+
+> 
+
+- **[Conference Site Project]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo available on Git HUB <a name="live-demo">https://baqar-abbas.github.io/Conference-site-Capstone-Project-Module1/</a>
+
+
+
+- [Live Demo Link](https://baqar-abbas.github.io/Conference-site-Capstone-Project-Module1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+ A New developer can view my project updates on Git HUB shared Repository
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+> Browser
+> GitHub Account
+> VS Code Editor
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder: 
+git clone <project repository link>
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+
+
+### Deployment
+
+ This project can be deployed using:
+Git Hub 
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project. (No Collaborators - Solo Capstone Project)
+Author - Baqar Abbas - Conference Page Project
+baqar007@hotmail.com available on Twitter and Linkedin profiles.
+
```

Comment:
- [ ] Good job adding the author section to your project however, the intention of the previous reviewer based on the guidelines given is to use the structure given on the README template. check the image below
- This is to give your work a professional view 

## View:
![image](https://user-images.githubusercontent.com/50990925/219527201-0c2b173b-18c9-4134-807e-478872a5a72f.png)

Reply:
Suggested changes have been incorporated within the project. Screen shot attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/219602737-d22e9a8a-e069-4824-a683-f551416add9d.png)

Reply:
![image](https://user-images.githubusercontent.com/117091923/219608964-748cf86c-bb9f-4c3c-8a78-bf1cba97f0f2.png)